config/crd/bases/rode.liatr.io_collectors.yaml

```diff
@@ -8,6 +8,10 @@ metadata:
   creationTimestamp: null
   name: collectors.rode.liatr.io
 spec:
+  additionalPrinterColumns:
```

Comment:
was this needed?  i thought Sean said these showed up "magically" by added the status.conditions

Reply:
The comment that resulted in this code is here: https://github.com/liatrio/rode/pull/10/files#diff-ad8159b186b7d71958135be8689c297eR45